src/web/nextui/src/app/eval/ResultsView.tsx

```diff
@@ -80,9 +81,17 @@ export default function ResultsView({
     evalId,
     inComparisonMode,
     setInComparisonMode,
+    columnStates,
+    setColumnState,
   } = useResultsViewStore();
   const { setStateFromConfig } = useMainStore();
 
+  const currentEvalId = evalId || defaultEvalId || 'default';
```

Comment:
instead of `|| 'default';` we could disable the feature. I think this only affects supabase but I am not sure.